config/locales/jobs.yml

```diff
@@ -100,13 +100,25 @@ en:
           <span class="govuk-!-font-weight-bold text-wrap-apostrophe text-capitalize">%{location}</span>
 
     sort_by:
+      expires_on:
```

Comment:
Some of the sort-bys use title case and others are all lower case. Could we align on one or the other (in consultation with @RoseMackworth)?

Either way, I think (Oldest) and (Soonest) will want to be lower case.

Reply:
yeah well picked up, i actually changed them to lowercase and changed them back again to align with figma and less friction (maybe laziness) biut you are right it should be one or the other and consistent

Reply:
confirmed everything should be lowercase